manifests/kube_addons.pp

```diff
@@ -29,7 +29,7 @@
   exec { 'Install cni network provider':
     command => "kubectl apply -f ${cni_network_provider}",
     onlyif  => 'kubectl get nodes',
-    creates => '/etc/cni/net.d'
+    unless  => "kubectl -n kube-system get daemonset | egrep '(flannel|weave)'"
```

Comment:
Can we also include calico-node in the daemonsets...as that is also a supported CNI network provider